Firestore/core/src/local/local_store.cc

```diff
@@ -465,6 +475,10 @@ void LocalStore::NotifyLocalViewChanges(
             target_data.WithLastLimboFreeSnapshotVersion(
                 last_limbo_free_snapshot_version);
         target_data_by_target_[target_id] = updated_target_data;
+
+        if (ShouldPersistTargetData(updated_target_data, target_data, {})) {
+          target_cache_->UpdateTarget(updated_target_data);
+        }
```

Comment:
isn't `updated_target_data` taking its last limbo-free snapshot version from `target_data`, and hence the difference between them will always be zero?